modules/speicher_fronius/main.sh

```diff
@@ -18,11 +17,11 @@ fi
 
 openwbDebugLog ${DMOD} 2 "Speicher IP: ${wrfroniusip}"
 
-bash "$OPENWBBASEDIR/packages/legacy_run.sh" "modules.fronius.device" "bat" "${wrfroniusip}" "${froniuserzeugung}" "0" "none" "" >>$MYLOGFILE 2>&1
+bash "$OPENWBBASEDIR/packages/legacy_run.sh" "modules.fronius.device" "bat" "${wrfroniusip}" "${froniuserzeugung}" "0">>"$MYLOGFILE" 2>&1
 ret=$?
 
 openwbDebugLog ${DMOD} 2 "RET: ${ret}"
 
-speicherleistung=$(<${RAMDISKDIR}/speicherleistung)
+speicherleistung=$(<"${RAMDISKDIR}"/speicherleistung)
```

Comment:
Auch wenn die Syntax korrekt ist, würde ich für eine bessere Lesbarkeit das schließende " ans Ende verschieben.
```suggestion
speicherleistung=$(<"${RAMDISKDIR}/speicherleistung")
```